src/legacy/core_plugins/kibana/public/index.scss

```diff
@@ -13,15 +13,15 @@
 // Discover styles
 @import 'discover/index';
 
-// Home styles
-@import './home/index';
-
 // Visualize styles
 @import './visualize/index';
 // Has to come after visualize because of some
 // bad cascading in the Editor layout
 @import 'src/legacy/ui/public/vis/index';
 
+// Home styles
+@import '../../../../plugins/home/public/application/index';
```

Comment:
Can you import this file directly in the JS in `plugins/home/public/` instead?

Reply:
And also then rename it from `_index.scss` to `index.scss`

Reply:
I originally included this but reverted because the scss is depending on legacy mixins in “ui/styles”. Importing from a module in the new platform gave me an error. The platform team is owning the topic and is tracking this separately, as soon as it’s moved the entry-point can be moved over just as you suggest.

Reply:
You could then just import the mixin dependencies directly in the `index.scss` file so as to unblock moving this to the JS file.

Reply:
It’s not complaining about the mixin itself (somehow it’s pulling that in implicitly), but about an asset used within the mixin: https://github.com/elastic/kibana/blob/master/src/legacy/ui/public/styles/_mixins.scss#L92

That’s the line failing. As it’s going to get migrated soon I didn’t dig any deeper , but maybe you know a better way around the problem.

Reply:
What does the error say?

Reply:
Also, I wonder then why yours would be the only implementation that complains if it's not the one pulling the file directly...

Reply:
I couldn't reproduce the problem locally anymore and the Jenkins build with this failure is already cleaned up - I committed the change again, if it works even better.

> Also, I wonder then why yours would be the only implementation that complains if it's not the one pulling the file directly...

There is no usage of that specific mixin from within the new platform so far, it's only consumed the legacy way:
```
x-pack/plugins/spaces/public/space_selector/_space_selector.scss
2:  @include kibanaFullScreenGraphics;

x-pack/legacy/plugins/security/public/components/authentication_state_page/_authentication_state_page.scss
2:  @include kibanaFullScreenGraphics;

x-pack/legacy/plugins/security/public/views/login/components/login_page/_login_page.scss
2:  @include kibanaFullScreenGraphics;

src/plugins/home/public/application/components/_welcome.scss
2:  @include kibanaFullScreenGraphics($euiZLevel6);
```

Spaces selector looks like it's imported in the new platform, but it's still used in the old way eventually: `x-pack/legacy/plugins/spaces/public/index.scss`

Reply:
I just pulled down your branch, ran it locally, didn't get any compile or console errors and the graphics rendered properly in the Welcome page. So... 👍 looks like all is good with the import in JS?